### PR TITLE
Set up config (with mocks) and test

### PR DESCRIPTION
**Pull request by cehsu**
Connects to department-of-veterans-affairs/vets.gov-team#3152
screen shot 2017-06-14 at 3 11 20 am
screen shot 2017-06-14 at 3 11 38 am

### CODE REVIEW

File: src/js/burials/config/form.js [hunk 1]
```diff
@@ -139,6 +147,34 @@ const formConfig = {
...
+    },
+    claimantContactInformation: {
+      title: 'Claimant Contact Information',
+      pages: {
+        claimantContactInformation: {
+          title: 'Claimant Contact Information',
+          path: 'claimant-contact-information',
+          uiSchema: {
+            'ui:validations': [
+              validateMatch('email', 'view:emailConfirmation')
```
Comment (jbalboni) [inline src/js/burials/config/form.js]: Do we need this?

File: test/burials/config/claimantContactInformation.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      <DefinitionTester
+          definitions={formConfig.defaultDefinitions}
+          schema={schema}
+          onSubmit={onSubmit}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = findDOMNode(form);
+
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_claimantAddress_street'), {
```
Comment (jbalboni) [inline test/burials/config/claimantContactInformation.unit.spec.jsx]: I added a helper to make this easier. In `schemaform-utils`, there's a `getFormDOM` function. If you replace `findDOMNode` with that, you can do `formDOM.fillData(id, value)`. There are examples in the marriage page tests for pensions.

(Don't feel like you have to convert this, but should make writing this kind of test easier in the future).

**Review by jbalboni — APPROVED** [review]
Looks good, had one question about the email validation, but that's it.

**Comment by cehsu** [comment]
Thanks very much for the review comments @jbalboni, changes made.

**Review by jbalboni — APPROVED** [review]

**Review by cvalarida — APPROVED** [review]

**Comment by J-Quag** [comment]
Did we want to have the email address check the same way we did it in Education?

**Comment by jbalboni** [comment]
@J-Quag Seems like a @goldenmeanie and @stacyhlinn question. It wasn't in the outline.

**Comment by stacyhlinn** [comment]
@jbalboni Let's make it consistent with the EDU form and add in an email address check. Thanks!

**Comment by cehsu** [comment]
@stacyhlinn if we are aiming for consistency, should we also have the same error message as in HCA? `Please put your email in this format x@x.xxx` or is it ok to stick with `Please enter a valid email address`?

**Comment by stacyhlinn** [comment]
@cehsu "Please enter a valid email address" seems more straightforward. If you agree, let's stick with it. :)

**Pull request merged**